core/src/test/scala/org/apache/spark/sql/test/SharedSQLContext.scala

```diff
@@ -345,9 +345,15 @@ trait SharedSQLContext
   private def initializeJDBCUrl(): Unit = {
     // TODO(Zhexuan Yang) for zero datetime issue, we need further investigation.
     //  https://github.com/pingcap/tispark/issues/1238
+
+    var SSLPara = "false"
+    if (conf.contains("jdbc.tls_enable") && conf.get("jdbc.tls_enable").equals("true")) {
+      SSLPara = "true&verifyServerCertificate=false&requireSSL=true"
```

Comment:
Hmm not verify client and server, then why include so many certs?

Reply:
This SSLPara is used for the JDCB client in the test data building. And JDBC certs will be injected into option.url that is used in write data test.
While JDBC connector can be successfully linked to TiDB server when it configures with certs or without certs.